src/components/button-toggle/button-toggle.scss

```diff
@@ -0,0 +1,46 @@
+@import "elevation";
+@import "default-theme";
+@import "palette";
+@import "mixins";
+
+$md-button-toggle-padding: 0 16px !default;
+$md-button-toggle-line-height: 36px !default;
+$md-button-toggle-border-radius: 3px !default;
+
+md-button-toggle-group {
+  @include md-elevation(2);
+  position: relative;
+  display: inline-flex;
+  border-radius: $md-button-toggle-border-radius;
+  cursor: pointer;
+  white-space: nowrap;
+}
+
+
+.md-button-toggle-checked .md-button-toggle-label-content {
+  background-color: md-color($md-grey, 300);
+}
+
+.md-button-toggle-disabled .md-button-toggle-label-content {
+  background-color: md-color($md-foreground, disabled);
+  cursor: not-allowed;
+}
+
+md-button-toggle {
+  white-space: nowrap;
+}
+
+.md-button-toggle-input {
+  @include md-visually-hidden;
+}
+
+.md-button-toggle-label-content {
+  display: inline-block;
+  line-height: $md-button-toggle-line-height;
+  padding: $md-button-toggle-padding;
+  cursor: pointer;
+}
+
+md-button-toggle md-icon {
+  vertical-align: middle;
```

Comment:
I don't have a strong opinion, I just wanted to be safe for this iteration when I wrote that.